doc/developer/design/00000000_template.md

```diff
@@ -1,99 +1,72 @@
-- Feature name: (Insert name of feature)
-- Associated: (Insert list of associated epics, issues, or PRs)
-
-# Summary
-[summary]: #summary
-
-One paragraph to explain the feature: What's wrong, what's the problem, what's the fix, what are the consequences?
-
-# Motivation
-[motivation]: #motivation
-
-Why are we doing this? What problems does it solve? What use cases does it enable? What is the desired outcome?
+# <Insert Name of Feature>
 
-# Explanation
-[explanation]: #explanation
-
-Explain the design as if it were part of Materialize and you were teaching the team about it.
-This can mean:
-
-- Introduce new named concepts.
-- Explain the feature using examples that demonstrate product-level changes.
-- Explain how it builds on the current architecture.
-- Explain how engineers and users should think about this change, and how it influences how everyone uses the product.
-- If needed, talk though errors, backwards-compatibility, or migration strategies.
-- Discuss how this affects maintainability, or whether it introduces concepts that might be hard to change in the future.
-
-# Reference explanation
-[reference-explanation]: #reference-explanation
-
-Focus on the implementation of the feature.
-This is the technical part of the design.
+- Associated: (Insert list of associated epics, issues, or PRs)
 
-- Is it reasonably clear how the feature is implemented?
-- What dependencies does the feature have and introduce?
-- Focus on corner cases.
-- How can we test the feature and protect against regressions?
+<!--
+Note: Feel free to add or remove sections as needed. However, most design
+docs should at least keep the suggested sections.
+-->
 
-# Rollout
-[rollout]: #rollout
+## Context & Scope
```

Comment:
Does "Scope" here refer to the scope of the proposed feature or something else? I'm asking because for the scope of the feature we already have the Goals and Non-Goals sections.

Reply:
For me at least, just "Context" would be perfect!